assets/dev/js/editor/components/browser-import/items/item.js

```diff
@@ -1,6 +1,13 @@
 import isInstanceof from '../../../utils/is-instanceof';
 import Mime from 'mime';
 
+/**
+ * @typedef {import('../files/file-parser-base')} FileParserBase
+ */
+/**
+ * @typedef {import('../files/file-reader-base')} FileReaderBase
+ */
+
```

Comment:
Please combine here and other `typedef`
```suggestion
/**
 * @typedef {import('../files/file-parser-base')} FileParserBase
 * @typedef {import('../files/file-reader-base')} FileReaderBase
 */

```